arches/app/views/api.py

```diff
@@ -365,6 +400,10 @@ def get(self, request, nodeid, zoom, x, y):
             raise Http404()
         return HttpResponse(tile, content_type="application/x-protobuf")
 
+    def create_mvt_cache_key(node, zoom, x, y, user, mvt_snapshot=None):
+        if not mvt_snapshot:
+            mvt_snapshot = models.EditLog.objects.filter(nodegroupid=str(node.nodegroup_id)).count()
```

Comment:
I'm not sure if this EditLog count is needed if you are setting the value of the tile in the cache to an empty string if no geometry was found (line 380), but maybe it has a different purpose that I'm overlooking.